src/apps/content-editor/src/app/views/ItemEdit/Content/Content.js

```diff
@@ -41,15 +58,27 @@ export default function Content(props) {
             onSave={props.onSave}
           />
         </div>
-        <div className={styles.Actions}>
-          <Actions
-            {...props}
-            site={{}}
-            set={{
-              type: props.model.type,
-            }}
-          />
-        </div>
+
+        <ActionsDrawer className={styles.Actions} {...props} />
+
+        <Button
```

Comment:
This seems like it would make more sense organizationally living in the `ActionsDrawer`